sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/implementation/Utility.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.http.netty.implementation;
+
+import io.netty.buffer.ByteBuf;
+import reactor.netty.Connection;
+
+import java.nio.ByteBuffer;
+
+/**
+ * Helper class containing utility methods.
+ */
+public final class Utility {
```

Comment:
Not a huge fan of how many Utility classes we have - can this be made into, e.g. NettyUtility or similar